dspace-api/src/main/java/org/dspace/content/authority/ChoiceAuthorityServiceImpl.java

```diff
@@ -184,10 +205,25 @@ public boolean isClosed(String fieldKey)
         }
         return null;
     }
+    
+    
+    @Override
+    public String getChoiceAuthorityName(String schema, String element, String qualifier) {
+    	String makeFieldKey = makeFieldKey(schema, element, qualifier);
+		if(getChoiceAuthorityMap().containsKey(makeFieldKey)) {			
+    		for(String key : this.authorities.keySet()) {
+    			if(this.authorities.get(key).equals(makeFieldKey)) {
```

Comment:
Please use the `containsValue` method here (https://docs.oracle.com/javase/7/docs/api/java/util/Map.html#containsValue(java.lang.Object))

Reply:
Noted. Thank you.

Reply:
Ok I missed that. This is OK then.